pkg/util/mon/bytes_usage.go

```diff
@@ -657,7 +683,7 @@ func (b *BoundAccount) Grow(ctx context.Context, x int64) error {
 		defer b.Mu.Unlock()
 	}
 	if b.reserved < x {
-		minExtra := b.mon.roundSize(x)
+		minExtra := b.mon.roundSize(x - b.reserved)
```

Comment:
hmm this was a bug?

Reply:
I dunno if it was a _bug_ per se?

Prior to the other changes here, `reserved` is typically very small, just a kb or whatever, so if you need to go to the parent for more, going for all you need vs all you need less what's already in `reserved` probably didn't matter to most callers. But my `Reserve()` api / earmarking change means that `reserved` should start to have some real non-trivial amounts in it now -- intentionally -- so ignoring it here becomes a bigger deal.